build/images/Dockerfile.agent.ubuntu

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 ARG BUILD_TAG
-FROM antrea/base-ubuntu:${BUILD_TAG}
+FROM antrea/base-ubuntu-${TARGETPLATFORM}:${BUILD_TAG}
```

Comment:
Should this be ${TARGETARCH}?

Reply:
Thanks. I was wondering why this was not caught in CI and during my testing, but then realized that this Dockerfile is not used in CI.